### ISSUE TITLE
Game crash on large teleport list

### ISSUE THREAD

**Issue by Rallymen007** [ac-custom-shaders-patch/acc-extension-config#111]
Hello!
(not sure this is the correct way of reporting a bug but here I go)

We're running a Glensheil server and I've been adding teleport locations to cope with falling through the map and the lack of a teleport to friend option but ran into an issue where after adding a couple more spawns the game started crashing after loading everything but before getting actually in game.
Investigated a little bit more and it seems that the issue is related to a server CSP extra config that becomes a bit too large.

Here's an extra config that crashes: [https://pastebin.com/RLHbPfCM](https://pastebin.com/RLHbPfCM)
And the same extra config, except I replaced " = " by "=" to make it a couple hundred bytes smaller and it loads fine [https://pastebin.com/nQQfw155](https://pastebin.com/nQQfw155)

Thanks!

**Comment by Marocco2** [comment]
Did you try latest builds? (0.1.76)

**Comment by Rallymen007** [comment]
Yeah and the bug still occurs, tested 0.1.76 with the same inis as in the pastebins.

**Comment by gro-ove** [comment]
Thanks, will be fixed in the next update! Sorry for taking so long

**Comment by Marocco2** [comment]
Still present in 0.1.77 public?
